lib/iso_builder.py

```diff
@@ -155,16 +155,11 @@ def _save(self):
                                   LATEST_SYMLINK_NAME)
         utils.force_symlink(self.timestamp, latest_dir)
 
-        iso_file = "%s-DVD-%s-%s.iso" % (self.distro, self.arch, self.version)
-        checksum_file = ("%s-%s-%s-CHECKSUM" %
-                         (self.distro, self.version, self.arch))
         iso_dir = "/%s/%s/iso" % (self.version, self.arch)
-        iso_path = os.path.join(iso_dir, iso_file)
-        checksum_path = os.path.join(iso_dir, checksum_file)
-        chroot_files = "%s %s" % (iso_path, checksum_path)
+        iso_files = "%s/*" % iso_dir
+        chroot_files = "%s" % iso_files
```

Comment:
You could simply remove the `chroot_files` variable and use `iso_files` everywhere.